packages/js/api/src/framework/__tests__/payload-transformer.spec.ts

```diff
@@ -0,0 +1,383 @@
+import {Order, BillingOrderAddress, ShippingOrderAddress, OrderLineItem, OrderShippingLine, OrderFeeLine, OrderTaxRate, OrderCouponLine, MetaData, OrderRefundLine} from "../../models";
```

Comment:
Once a linter is implemented, it will complain about the line length. Typically, if it's more than 3 or 4 objects we format these one per line.

Reply:
Technically, I think this package already has linting via `pnpm nx lint @woocommerce/api`, or just `pnpm run lint`, but our CI doesn't execute tests or linting in packages.

Reply:
fwiw, the linting threw 2k errors, of which 1700 are automatically fixable with `npm run lint --fix`.

Of the remaning 300, I'd say 250 are `Expected preceding "Internal dependencies" comment block  @woocommerce/dependency-group`, which we can ignore, and the remaining 50 would have to be looked over manually. I'll add an issue to the backlog to run linting on the `@woocommerce/api` package in CI, as it helps with reducing code reviews for code style, and minimizes merge conflicts.

Reply:
Line breaks added for those lengthy imports! https://github.com/woocommerce/woocommerce/pull/31665/commits/ae5d23ce9f6148c1bb2bae8c08fffa846373fb9b